src/devices/sound/ad1848.cpp

```diff
@@ -50,7 +50,7 @@ void ad1848_device::device_start()
 
 void ad1848_device::device_reset()
 {
-	memset(&m_regs.idx[0], '\0', sizeof(m_regs));
+	memset(m_regs.idx, '\0', sizeof(m_regs.idx));
```

Comment:
I'd prefer `std::fill(std::begin(foo), std::end(foo), 0)` for this kind of thing - it will continue to work if the type of container changes, and it will throw a fit if it's changed to a pointer rather than doing the wrong thing.